dev-docs/modules/zeusPrimeRtdProvider.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: page_v2
+title: Zeus Prime Real Time Data Provider
+display_name: Zeus Prime RTD
+description: Zeus Prime Real Time Data Module
+page_type: module
+module_code: zeusPrimeRtdProvider
+module_type: rtd
+enable_download: true
+vendor_specific: true
+sidebarType: 1
+---
+
+# Zeus Prime Real Time Data Module
+
+The Zeus Prime RTD Provider provides integration of Zeus Prime onto sites with Prebid. This module will request information from Zeus Prime servers to add the page level targeting required for Prime into the customer's ad setup.
+
+Zeus Prime runs as soon as the code is initialized, so it can retrieve the information required from the Zeus Prime server to create the targeting key-values. Zeus Prime will provide two page level key-values: `zeus_<yourGamId>` and `zeus_insights`. Zeus Prime provides contextual information about a pages content, and does not provide user information that could present privacy implications.
+
+For more information and help with setting up Zeus Prime, see the [onboarding documentation site](https://onboarding.zeustechnology.com).
+
+## Usage
+
+To use Zeus Prime, add `zeusPrimeRtdProvider` into your Prebid build:
+
+```
+gulp build --modules=rtdModule,zeusPrimeRtdProvider
+```
+
+> Note that the global RTD module, `rtdModule`, is required for the Zeus Prime RTD module.
+
+Once the code is included, configure Zeus Prime in your Prebid configuration:
+
+```javascript
+pbjs.setConfig({
+  ...,
+  realTimeData: {
+    dataProviders: [{
+      name: 'zeusPrime',
+      params: {
+        gamId: '<yourGamId>'
+      }
+    }]
+  },
+  ...
+})
+```
+
+## Parameters
+
+The parameters below describe the configuration object used to configure Zeus Prime.
+
+| Name         | Type    | Description                                                             | Default   |
+| ------------ | ------- | ----------------------------------------------------------------------- | --------- |
+| name         | String  | This will always be `zeusPrime`                                         | -         |
+| waitForIt    | Boolean | Set to false. Zeus Prime runs on load and not during the bidding cycle. | `default` |
```

Comment:
Can you change the heading of the last column from Default to Notes please? Then something like defaults to false so the first part of wait for it is in the notes instead of the description 🙏

Reply:
You got it! Cleaned that up a bit too to based on other modules/documentation.